### PR TITLE
Add HOLDDATA for issue 1511

### PR DESCRIPTION
**Pull request by John-A-Davies**
Signed-off-by: John Davies <daviesja@uk.ibm.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Necessary documentation (if appropriate) have been added / updated
- [ ] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [x] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes  #1511 

#### Changes proposed in this PR

- 

Adds HOLDDATA.

#### Does this PR introduce a breaking change?

- [X] Yes
- [ ] No

#### Does this PR do something the person installing Zowe should know about?

This is the HOLDDATA for "Configure external and referrer hostnames"

---
* Affected function: _general area of interest_ *

configuration.

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

instance.env

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number: 

#### Other information

This is for a USERMOD.

### CODE REVIEW

**Comment by OnnoVdT** [comment]
does this HOLD info also need to go in 1.14 ? If so I will update PR #1538

**Comment by John-A-Davies** [comment]
> 
> 
> does this HOLD info also need to go in 1.14 ? If so I will update PR #1538

Yes, I think all USERMOD HOLDDATA (like this one) should be rolled up into the HOLDDATA for the next release.

File: smpe/bld/service/current-hold-ACTION.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+or:
+
+Define ZWE_REFERRER_HOSTS the same way, for the value to be 
+applied specifically for referrer purposes.
+
+New variables are added to instance.env :
+  
+- ZWE_EXTERNAL_HOSTS=host1,host2,...
+# comma-separated list of hosts
```
Comment (John-A-Davies) [inline smpe/bld/service/current-hold-ACTION.txt:31]: Should this line start with '*' instead of '#'?

Comment (OnnoVdT) [reply smpe/bld/service/current-hold-ACTION.txt:31]: no, instance.env is read like a shell script, so use shell-style comments

Comment (John-A-Davies) [reply smpe/bld/service/current-hold-ACTION.txt:31]: Sure, as long as the HOLDDATA parser does not complain.

Comment (OnnoVdT) [reply smpe/bld/service/current-hold-ACTION.txt:31]: I added this hold data to PR #1538, so I think this PR can be closed. If we want to keep it around we must update it to NOT remove the ptf-bucket, or the PTF build will fail.
Build 5 of #1538 is an SMPE build that will have the hold info.

Comment (OnnoVdT) [reply smpe/bld/service/current-hold-ACTION.txt:31]: as fold hold data parsing, round brackets ( ) must be paired. and you cannot use /* */ as SMPE sees this as comments (troublesome for adding sample JCL which can hold //*)

Comment (John-A-Davies) [reply smpe/bld/service/current-hold-ACTION.txt:31]: Noted, thank.  But I don't see any of those deprecated usages in this file, so that's OK, right?

Comment (OnnoVdT) [reply smpe/bld/service/current-hold-ACTION.txt:31]: Not sure I follow. What do you mean with deprecated usages?

Comment (John-A-Davies) [reply smpe/bld/service/current-hold-ACTION.txt:31]: I mean I don't see any
`/* ... */`
`( ... )`
Conclusion:  Your file is OK as it now is.

**Comment by John-A-Davies** [comment]
I can confirm that https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-1538/2/consoleFull does contain the HOLDDATA.
In particular, @1000TurquoisePogs it contains the info you requested, shown below:
```
                                            ****************************************************************
                                            * Timing: post-APPLY                                           *
                                            ****************************************************************
                                            * Part: instance.env                                           *
                                            ****************************************************************
                                            If you are upgrading to Zowe 1.14 or higher from a previous
                                            release, and the value of ZOWE_EXPLORER_HOST does not match the
                                            host and domain that you put into your browser to access Zowe,
                                            then you must update your configuration due to updated
                                            referrer-based security.
  
                                            You can either:
  
                                            Define ZWE_EXTERNAL_HOSTS equal to a comma-separated list of
                                            hosts names you use to access Zowe from your browser,
  
                                            or:
  
                                            Define ZWE_REFERRER_HOSTS the same way, for the values to be
                                            applied specifically for referrer purposes.
  
                                            New variables are added to instance.env :
  
                                            ZWE_EXTERNAL_HOSTS=host1,host2,...
                                            # comma-separated list of hosts names you use to access Zowe
                                            # from your browser
  
                                            ZWE_REFERRER_HOSTS=host1,host2,...
                                            # comma-separated list of hosts names you use to access Zowe
                                            # from your browser, where the value is to be applied
                                            # specifically for referrer purposes
```

**Comment by OnnoVdT** [comment]
so can we cancel this PR now ?

**Pull request closed without merging**